### PR TITLE
103 convert paddle to OV without legacy ONNX path

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by ryanloney** [comment]
I'm not able to suggest an edit for some reason on the beginning of notebook, but top of the .ipynb says "Convert a PaddlePaddle Model to ONNX and OpenVINO™ IR" Need to remove the ONNX part @eaidova

File: notebooks/103-paddle-to-openvino/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ # PaddlePaddle to ONNX and OpenVINO™ IR Tutorial
```
Comment (ryanloney) [inline notebooks/103-paddle-to-openvino/README.md:1]:
```suggestion
 # PaddlePaddle to OpenVINO™ IR Tutorial
```

File: README.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ More amazing notebooks here!
...
 | [101-tensorflow-to-openvino](notebooks/101-tensorflow-to-openvino/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F101-tensorflow-to-openvino%2F101-tensorflow-to-openvino.ipynb) | Convert TensorFlow models to OpenVINO IR | 
 | [102-pytorch-onnx-to-openvino](notebooks/102-pytorch-onnx-to-openvino/) | Convert PyTorch models to OpenVINO IR | 
-| [103-paddle-onnx-to-openvino](notebooks/103-paddle-onnx-to-openvino/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F103-paddle-onnx-to-openvino%2F103-paddle-onnx-to-openvino-classification.ipynb) | Convert PaddlePaddle models to OpenVINO IR | 
+| [103-paddle-to-openvino](notebooks/103-paddle-onnx-to-openvino/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F103-paddle-to-openvino%2F103-paddle-to-openvino-classification.ipynb) | Convert PaddlePaddle models to OpenVINO IR |
```
Comment (ryanloney) [inline README.md:90]:
```suggestion
| [103-paddle-to-openvino](notebooks/103-paddle-to-openvino/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F103-paddle-to-openvino%2F103-paddle-to-openvino-classification.ipynb) | Convert PaddlePaddle models to OpenVINO IR | 
```

**Review by ryanloney — CHANGES_REQUESTED** [review]
Looks good except for a few minor changes (ONNX still showing up in a README, top of ipynb and one of the URL in README)

**Comment by eaidova** [comment]
> I'm not able to suggest an edit for some reason on the beginning of notebook, but top of the .ipynb says "Convert a PaddlePaddle Model to ONNX and OpenVINO™ IR" Need to remove the ONNX part @eaidova

@ryanloney  thank you, updated

**Review by adrianboguszewski — COMMENTED** [review]
According to the contributing guide, any image should go to "data" dir.

**Comment by eaidova** [comment]
> According to the contributing guide, any image should go to "data" dir.

I just rename directory, image was located in the current dir in main branch https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/103-paddle-onnx-to-openvino/coco_close.png

**Comment by adrianboguszewski** [comment]
> > According to the contributing guide, any image should go to "data" dir.
> 
> I just rename directory, image was located in the current dir in main branch https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/103-paddle-onnx-to-openvino/coco_close.png

Ok, I will create separate issue for that.

**Comment by eaidova** [comment]
> > > According to the contributing guide, any image should go to "data" dir.
> > 
> > 
> > I just rename directory, image was located in the current dir in main branch https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/103-paddle-onnx-to-openvino/coco_close.png
> 
> Ok, I will create separate issue for that.

I moved image

**Comment by eaidova** [comment]
@ryanloney do you have any additional comments?

File: notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb [hunk 3]
```diff
@@ -52,9 +53,10 @@
    "source": [
     "### Settings\n",
     "\n",
-    "Set `IMAGE_FILENAME` to the filename of an image to use. Set `MODEL_NAME` to the PaddlePaddle model to download from PaddleHub. `MODEL_NAME` will also be the base name for the converted ONNX and IR models. The notebook is tested with the [mobilenet_v3_large_imagenet_ssld](https://www.paddlepaddle.org.cn/hubdetail?name=mobilenet_v3_large_imagenet_ssld&en_category=ImageClassification) model. Other models may use different preprocessing methods and therefore require some modification to get the same results on the original and converted model.\n",
+    "Set `IMAGE_FILENAME` to the filename of an image to use. Set `MODEL_NAME` to the PaddlePaddle model to download from PaddleHub. `MODEL_NAME` will also be the base name for the IR model. The notebook is tested with the [mobilenet_v3_large_x1_0](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/en/models/Mobile_en.md) model. Other models may use different preprocessing methods and therefore require some modification to get the same results on the original and converted model.\n",
```
Comment (ryanloney) [inline notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb:56]:
```suggestion
    "Set `IMAGE_FILENAME` to the filename of an image to use. Set `MODEL_NAME` to the PaddlePaddle model to download from PaddleHub. `MODEL_NAME` will also be the base name for the IR model. The notebook is tested with the [mobilenet_v3_large_x1_0](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/en/models/Mobile_en.md) model. Other models may use different preprocessing methods and therefore require some modification to get the same results on the original and converted model.\n",
```

File: notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb [hunk 4]
```diff
@@ -77,9 +89,7 @@
    "source": [
     "## Show Inference on PaddlePaddle Model\n",
     "\n",
-    "In the next cell, we load and download the model from PaddleHub, read and display an image, do inference on that image, and show the top three prediction results.\n",
-    "\n",
-    "The first time you run this notebook, the PaddlePaddle model is downloaded from PaddleHub. This may take a while."
+    "In the next cell, we load model, read and display an image, do inference on that image, and show the top three prediction results."
```
Comment (ryanloney) [inline notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb:92]:
```suggestion
    "In the next cell, we load the model, load and display an image, do inference on that image, and then show the top 3 prediction results."
```

File: notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb [hunk 5]
```diff
@@ -104,7 +114,7 @@
...
+    "`classifier.predict()` takes an image file name, reads image and preprocesses input and returns the class labels and scores of the image. Preprocessing the image is done behind the scenes. The classification model returns an array with floating point values for each of the 1000 ImageNet classes. The higher the value, the more confident the network is that the class number corresponding to that value (the index of that value in the network output array) is the class number for the image. \n",
```
Comment (ryanloney) [inline notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb:117]:
```suggestion
    "`classifier.predict()` takes an image file name, reads the image, preprocesses the input, then returns the class labels and scores of the image. Preprocessing the image is done behind the scenes. The classification model returns an array with floating point values for each of the 1000 ImageNet classes. The higher the value, the more confident the network is that the class number corresponding to that value (the index of that value in the network output array) is the class number for the image. \n",
```

File: notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb [hunk 6]
```diff
@@ -122,20 +132,19 @@
...
+    "# classifier.get_config()"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "ec080a4d",
    "metadata": {},
    "source": [
-    "The `data_feed` module shows that images are normalized, resized and cropped and that the BGR image is converted to RGB before propagating through the network. In the next cell, we import the `process_image` function that is defined in this file to do inference on the OpenVINO IR model with the same method. "
+    "The `classifier.get_config()` module shows preprocewssing configuration for model. it should be seen that images are normalized, resized and cropped and that the BGR image is converted to RGB before propagating through the network. In the next cell, we get the `classifier.predictror.preprocess_ops` propertiy that returns list of preprocessing operation to do inference on the OpenVINO IR model with the same method. "
```
Comment (ryanloney) [inline notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb:147]:
```suggestion
    "The `classifier.get_config()` module shows the preprocessing configuration for the model. It should show that images are normalized, resized and cropped, and that the BGR image is converted to RGB before propagating it through the network. In the next cell, we get the `classifier.predictror.preprocess_ops` property that returns list of preprocessing operations to do inference on the OpenVINO IR model using the same method. "
```

File: notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb [hunk 7]
```diff
@@ -164,77 +179,42 @@
...
-  },
-  {
-   "cell_type": "markdown",
-   "id": "a822664b",
-   "metadata": {},
-   "source": [
-    "### Preparation\n",
-    "\n",
-    "PaddlePaddle's MobileNet model contains information about the input shape, mean and scale values that we can use to convert the model. The next cell shows how to get these values."
+    "For decoding lables predicted by model to names of classes, we need to have mapping between them, model config contains information about `class_id_map_file`, which store such mapping. The code below, shows how to parse it to dictionary for using with openvino model."
```
Comment (ryanloney) [inline notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb:193]:
```suggestion
    "To decode the labels predicted by the model to names of classes, we need to have a mapping between them. The model config contains information about `class_id_map_file`, which stores such mapping. The code below shows how to parse the mapping into a dictionary to use with the OpenVINO model."
```

**Review by ryanloney — CHANGES_REQUESTED** [review]
@eaidova please see my suggested edits

**Review by ryanloney — APPROVED** [review]

**Pull request merged**